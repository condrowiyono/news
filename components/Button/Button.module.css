.button {
  border: none;
  background: #ED2227;
  box-shadow: 8px 8px 16px rgba(237, 34, 50, 0.16);
  border-radius: 8px;
  width: 100%;
  text-align: center;
  color: #FFFFFF;
  padding-top: 16px;
  padding-bottom: 16px;
  font-weight: 700;
  font-size: 14px;
  cursor: pointer;
}

.button:hover {
  background: #bd1c1f;
  transition: all 0.5s;
}