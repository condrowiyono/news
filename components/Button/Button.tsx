import style from "./Button.module.css";

const Button = () => {
  return (
    <button className={style.button} data-testid="button">
      Follow
    </button>
  );
};

export default Button;
