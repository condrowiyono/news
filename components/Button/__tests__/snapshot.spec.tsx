import { render } from "@testing-library/react";
import Button from "@components/Button";

it("renders Button unchanged", () => {
  const { container } = render(<Button />);
  expect(container).toMatchSnapshot();
});
