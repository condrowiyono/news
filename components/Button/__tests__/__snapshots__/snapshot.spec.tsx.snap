// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button unchanged 1`] = `
<div>
  <button
    class="button"
    data-testid="button"
  >
    Follow
  </button>
</div>
`;
